yaml/pois.yaml

```diff
@@ -562,14 +562,34 @@ filters:
       tier: 3
   # university
   - filter: {amenity: university}
-    min_zoom: { min: [ { max: [ { sum: [ { col: zoom }, 2.55 ] }, *tier3_min_zoom ] }, 16 ] }
+    min_zoom:
+      lookup:
+        key: { col: way_area }
+        op: '>='
+        table:
+          - [ 10, 10000000 ]
+          - [ 11,  5000000 ]
+          - [ 12,  1000000 ]
+          - [ 13,   500000 ]
+          - [ 14,   500000 ]
+          - [ 15,   100000 ]
+        default: 16
     output:
       <<: *output_properties
       kind: university
       tier: 3
   # college
   - filter: {amenity: college}
-    min_zoom: { min: [ { max: [ { sum: [ { col: zoom }, 2.35 ] }, *tier3_min_zoom ] }, 16 ] }
+    min_zoom:
+      lookup:
+        key: { col: way_area }
+        op: '>='
+        table:
+          - [ 12,   600000 ]
```

Comment:
Similar comment to https://github.com/tilezen/vector-datasource/pull/1763/files#r244405058, though this one used a different _scale factor_ of **2.35** versus **2.55** so maybe that's why the values are less as proposed?

Reply:
I suspect we can just reuse:

```
- [ 12,   500000 ]
- [ 13,   100000 ]
- [ 14,    50000 ]
- [ 15,     2000 ]
```

Reply:
OK, thanks. Updated in 5af8d31.